src/stylesheets/_forms_actions.scss

```diff
@@ -22,6 +22,6 @@
     justify-content: center;
 }
 
```

Comment:
It would be nice to add a class here for .form-actions--right

Reply:
added